### PR TITLE
Update to system.commandline

### PR DESCRIPTION
**Pull request by cartermp**
I hate so much the giant baby

### CODE REVIEW

File: src/try-convert/Program.cs [hunk 1]
```diff
@@ -15,24 +15,42 @@ internal class Program
     {
         private static async Task<int> Main(string[] args)
         {
-            var parser = new CommandLineBuilder(new Command("try-convert", handler: CommandHandler.Create(typeof(Program).GetMethod(nameof(Run)))))
-                .UseParseDirective()
-                .UseHelp()
-                .UseDebugDirective()
```
Comment (jmarolf) [inline src/try-convert/Program.cs:21]: is all the rest of this automatic now?

File: src/try-convert/Program.cs [hunk 2]
```diff
@@ -15,24 +15,42 @@ internal class Program
...
+                new Option(new[] { "--no-backup" }, "Converts projects and does not create a backup of the originals.")
+                {
+                    Argument = new Argument<bool>(() => false)
+                }
+            };
+
+            rootCommand.Description = "try-convert";
+            rootCommand.Handler = CommandHandler.Create(typeof(Program).GetMethod(nameof(Run)));
 
-            return await parser.InvokeAsync(args.Length > 0 ? args : new string[] { "-h" }).ConfigureAwait(false);
```
Comment (jmarolf) [inline src/try-convert/Program.cs:35]: This breaks showing help if the user does not pass in anything

File: src/try-convert/Program.cs (hunk 1 above)
Comment (cartermp) [reply src/try-convert/Program.cs:21]: nope

**Comment by cartermp** [comment]
image

I guess I should test everything else, though I must admit that I have never made use of the following outside of `UseHelp`:

```
                .UseParseDirective()
                .UseDebugDirective()
                .UseSuggestDirective()
                .RegisterWithDotnetSuggest()
                .UseParseErrorReporting()
                .UseExceptionHandler()
```

**Comment by cartermp** [comment]
rebase-a-rooed

**Pull request merged**